apps/prairielearn/src/sync/course-db.ts

```diff
@@ -876,6 +877,12 @@ async function loadInfoForDirectory<T extends { uuid: string }>({
         tolerateMissing: recursive,
       });
       if (info) {
+        if (info.data) {
+          //it's a draft if it's of the form .../__drafts__/question/info.json
+          //only Questions have property draft or clientFiles
+          const pathSplit = infoFilePath.split(path.sep);
+          info.data['draft'] = pathSplit[pathSplit.length - 3] === '__drafts__';
+        }
```

Comment:
This will run for many types of JSON files, not just questions. IMO this belongs in `loadQuestions` instead.

Reply:
moved in 48c7d4c7ffcb4e124ac714e8c07e2c896df9260c.